src/Games/NexusMods.Games.AdvancedInstaller.UI/Content/BodyView.axaml.cs

```diff
@@ -14,12 +15,21 @@ public BodyView()
 
         this.WhenActivated(disposables =>
         {
-            ModNameTextBlock.Text = ViewModel!.ModName.ToUpper();
+            // Set the mod name if VM is not null.
+            this.WhenAnyValue(view => view.ViewModel)
+                .WhereNotNull()
+                .Do(vm => ModNameTextBlock.Text = vm.ModName.ToUpper())
+                .Subscribe()
+                .DisposeWith(disposables);
 
-            this.OneWayBind(ViewModel, vm => vm.ModContentViewModel, view => view.ModContentSectionViewHost.ViewModel!)
+            // Bind the mod content view model.
+            this.OneWayBind(ViewModel, vm => vm.ModContentViewModel,
+                    view => view.ModContentSectionViewHost.ViewModel!)
                 .DisposeWith(disposables);
 
-            this.OneWayBind(ViewModel, vm => vm.CurrentPreviewViewModel, view => view.PreviewSectionViewHost.ViewModel!)
+            // Bind the right content view model.
+            this.OneWayBind(ViewModel, vm => vm.CurrentRightContentViewModel,
+                    view => view.PreviewSectionViewHost.ViewModel!)
                 .DisposeWith(disposables);
```

Comment:
The `!` at the end of `view => /* omitted */.ViewModel!` does nothing and can be removed.